### ISSUE TITLE
can't access the dataset 

### ISSUE THREAD

**Issue by ghost** [nikhil-salodkar/facial_expression#3]
Hi,
Thanks for sharing your implementation. I tried to download the dataset multiple times, filled out the request form long ago but haven't heard back. Can you please upload the dataset somewhere that we can have access to? I really appreciate it.

**Comment by nikhil-salodkar** [comment]
Hi,

Unfortunately, I lost that dataset because the hard disk where I stored it
got corrupted.

Nikhil

**Comment by danielrosero** [comment]
> Hi,
> Thanks for sharing your implementation. I tried to download the dataset multiple times, filled out the request form long ago but haven't heard back. Can you please upload the dataset somewhere that we can have access to? I really appreciate it.

You can find it here: https://github.com/ankitsharma285/AffectNet

**Comment by sanjeelparekh** [comment]
Hi,
In urgent need of this dataset!
@zahra-anvari  - were you able to access it? From the official website - looks like there are difficulties/delays in sharing due to multiple requests. Also, the registration form is no longer available.
@danielrosero - unfortunately, the link you shared returns a 404 error now! Any other ways of downloading?

I'll be grateful if any of  you could suggest ways for me to access this dataset.
Thanks

**Comment by danielrosero** [comment]
> Hi,
> In urgent need of this dataset!
> @zahra-anvari - were you able to access it? From the official website - looks like there are difficulties/delays in sharing due to multiple requests. Also, the registration form is no longer available.
> @danielrosero - unfortunately, the link you shared returns a 404 error now! Any other ways of downloading?
> 
> I'll be grateful if any of you could suggest ways for me to access this dataset.
> Thanks

I just googled it, and found another repo:
https://github.com/sealhuang/AffectNet

Hope this helps.

**Comment by sanjeelparekh** [comment]
@danielrosero Thanks a lot for the link. I suppose it doesn't contain the whole dataset, a good start nonetheless.

**Comment by danielrosero** [comment]
> @danielrosero Thanks a lot for the link. I suppose it doesn't contain the whole dataset, a good start nonetheless.

@sanjeelparekh Yeah is a shame. Needed the full dataset as well for my master's thesis, but found something similar to this one. I think is 10% of the 420k images official one.

Have fun!
